econplayground/templates/main/cohort_password_form.html

```diff
@@ -10,8 +10,9 @@ <h1>Password Required</h1>
     </p>
     <form class="form-group" method="post" action=".">{% csrf_token %}
         <div class="form-group">
+            <label for="password">Password</label>
             <input type="password" class="form-control"
-                   placeholder="Password" name="password" />
+                   placeholder="Password" id="password" name="password" />
```

Comment:
Because DOM id's need to be unique, I would make this one a bit more specific, maybe `cohort-password`